activejob/lib/active_job/queue_adapters/sqs_adapter.rb

```diff
@@ -0,0 +1,38 @@
+require 'shoryuken'
+
+module ActiveJob
+  module QueueAdapters
+    # == Amazon Simple Queue Service adapter for Active Job
+    #
+    # Super-efficient AWS SQS thread-based message processor. for Ruby.
+    #
+    # Read more about SQS {here}[https://aws.amazon.com/sqs/].
+    #
+    # To use SQS set the queue_adapter config to +:sqs+.
+    #
+    #   Rails.application.config.active_job.queue_adapter = :sqs
+    class SqsAdapter
+      def enqueue(job) #:nodoc:
+        JobWrapper.queue_as job.queue_name
+        ActiveSupport::JSON.encode(job.serialize)
```

Comment:
@seuros It's just some testing i was doing, but before i go ahead and complete this feature, i wanna know what others think about having Amazon SQS as adapter.